### PR TITLE
Open-sourcing reduced-bias BlenderBot variants

### PR DESCRIPTION
**Pull request by EricMichaelSmith**
Code and documentation for open-sourcing reduced-bias versions of BlenderBot3B.

### CODE REVIEW

**Review by stephenroller — APPROVED** [review]
A GPU test may be worthwhile

**Comment by EricMichaelSmith** [comment]
> A GPU test may be worthwhile

Good point, added

**Review by stephenroller — APPROVED** [review]
ty

File: parlai/zoo/model_list.py [hunk 1]
```diff
@@ -2075,4 +2075,92 @@
             "[TransformerGenerator]: I want to adopt a corgi puppy."
         ),
     },
+    {
+        "title": "BlenderBot3B with name-scrambling gender-bias reduction",
+        "id": "dialogue_bias",
+        "path": "zoo:dialogue_bias/gender__name_scrambling/model",
+        "agent": "transformer/generator",
+        "task": "blended_skill_talk",
+        "project": "https://github.com/facebookresearch/ParlAI/tree/master/projects/dialogue_bias",
```
Comment (stephenroller) [inline parlai/zoo/model_list.py:2084]: This is wrong, it's `main` now

**Pull request merged**